### DISCUSSION TITLE
[Feature] Remove Duplicates - Option to Archive or Stack, and UI tweaks

### DISCUSSION THREAD

**Discussion by pjft** [immich-app/immich#10148] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The new "Remove Duplicates" feature is great, but doesn't really work for read-only external libraries. Any chance we could get the options to Archive and/or Stack instead of Trash for the duplicate files? 

Also, from a UI perspective, having a "Keep all" button from the get go would go a long way, rather than having the user select all assumed duplicates and then select "Keep all". This happens a lot in burst photos.

Thanks!

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by brfransen** [comment]
I love the duplicates feature too.

It would be helpful if the date/time and location could be shown in the comparison UI.

**Comment by Hely0n** [comment]
I just wanted to create a Feature Request for this too!

Stacking similar images that were detected as duplicates would be awesome.

This would effectively solve an idea I posted once:
#6521

**Comment by Vcele** [comment]
It would be great I you could open the photo in full view

**Comment by genebean** [comment]
This is basically what I'd like to see too. I have just imported “all the things” to Immich and have nearly 11,000 duplicates, according to the app. I would like to be able to move all the ones that would currently be deleted to an archive or other hidden-from-gallery-by-default place so that they were out of sight, but not gone. This would allow me to hit one button to deal with all 11,000 and know that I could go looking in the duplicates album if I find I’m missing something later.

**Reply by sammyke007** [reply]
+1, an option to stack all at once!

**Comment by thekoos** [comment]
I do astro and stacked landscape photography which requires 100s of exposures of the exact same thing, which are picked up as duplicates (mostly). Not all images make it into to the final stacked images, so I do not want to mark the ones left out as "Stacked". These images are important so I do not want to delete them. Being able to Archive directly from the Duplicate Finder tool will save many hours of cleanup. Just let the options be Trash/Stack/Archive right from the duplicate finder screen.

**Comment by bo0tzz** [comment]
#11114
